### ISSUE TITLE
Missing https://new-raku.finanalyst.org/syntax/rule

### ISSUE THREAD

**Issue by coke** [Raku/doc-website#86]
https://docs.raku.org/syntax/rule exists, expect all the URLS in the old site to work in the new. (added .html just in case, fails either way)

**Comment by finanalyst** [comment]
This seems to be a corner case I haven't been aware of. It'll need some time to work on.

Are there any others. I had not thought to compare the rendered html files generated by Documentable and those generated by Collection, that should pick up files generated by one and not generated by the other.

**Comment by finanalyst** [comment]
@coke Can you point me to where the static html files are located? Is there are tar ball of the html files.

**Comment by dontlaugh** [comment]
I don't know how to get this without re-doing the previous documentable compilation process. But it occurs to me that before we go live with the new site, we should be sure to take a tarball archive backup of all the current site's assets.

**Comment by coke** [comment]
@dontlaugh  - can you get @finanalyst a tarball of the current docs.raku.org for comparison?

**Comment by finanalyst** [comment]
@coke 
# tl;dr 
I have looked at the original POD6 source.
a) The source is highly non-standard
b) Its actually very unclear to me how this file is even generated by `Documentable`, and if I don't understand it, I wonder how many other people today would.
c) No link to `syntax/rule` seems to exist in the documentation or `.htacess`

@coke How did you discover this file?

Suggestion: this seems such a corner case, I suggest we move the issue from GoLive to Cleanup.

# Detail
Secondary files are created by parsing the text of `=headx` Pod::Blocks through a grammar and picking out keywords such as `prefix` `method` `routine`. The rest of the source to the next `=head` is then copied into a new file together with the occurrences of the same keyword in other sources. 

The source for the file at issue is in `Type/Grammars.pod6` and is 
```
=head2 X<Rules|Syntax,rule>
```

## inconsistent markup
As you can see, there is no ambient text. Instead the whole title is enclosed in a `X<>` markup.

This is very inconsistent with other titles in the documentation. It indicates that the documentation was written very early on in the documentation process.

Text in Headers is already picked up for inclusion in a Table of Contents, so marking the same text for inclusion in a Glossary is overkill.

## puzzling origination

Quite how `Syntax,rule` is picked out, and not `Rules` is confusing, as the `recurse-to-str` sub (the code is common to both Documentable and Colllection) should only pick up `Rules`. 

In addition, none of the regexes in the grammar used to parse the header should match either `Rules`, or `Syntax,rule`. So I cant even see how the secondary code is even generated.

## no link in documentation

Looking through all the errors in [more->anomalous links](https://new-raku.finanalyst.org/error-report.html), there is no entry for `syntax/rule`. This means that it wasn't registered as a link in the Documentation as it currently exists.

@coke How did you discover the file in the first place?

**Comment by finanalyst** [comment]
@coke I think we can eliminate this issue from doc-website. It seemed odd in the Documentable build, its even odder here. I actually don't understand the issue that JJ opened.
@Altai-man also found it strange.
Like I said above, there is no Link I can find in the existing Documentation that references this web page. 
I suggest we close this issue.

**Comment by Altai-man** [comment]
Well, technically it is rendered: http://164.90.207.89:10010/syntax/rule

@finanalyst understanding of how grammar creating secondary pages is correct, I understand it as well (related to "how many people issue"). While I will gladly vote for secondary pages being generated for routines, when it comes to language pages, I'd consider them less useful myself, so I don't think it's a big deal. In short: remove it.

**Comment by finanalyst** [comment]
@Altai-man I thought you probably were one of the only people :)
You changed my mind about secondaries in general, so I put the work in to generate them. 
However, `syntax/rule` is not generated by Collection, and since the regex rules are the same, I actually don't know how Documentable `Doc::Page::Secondary` does generate them. Working out why seems more effort than its worth.
So, we can close this issue for the time being
